.github/workflows/check-performance-regression.yml

```
@@ -22,7 +22,16 @@ concurrency:
 jobs:
   test:
     if: github.repository == 'checkstyle/checkstyle'
+    strategy:
+      matrix:
+        grammar:
+          - java
+          - javadoc
     runs-on: ubuntu-latest
+    env:
+      BASELINE_SECONDS: ${{ matrix.grammar == 'java' && 384.02 || 530 }}
+      CONFIG_FILE: ${{ matrix.grammar == 'java' && './config/benchmark-config.xml'
+                                    || './config/benchmark-javadoc-config.xml' }}
```

Please do something like:
```suggestion
strategy:
  matrix:
    target:
      - java
      - javadoc
    include:
      - target: java
        BASELINE_SECONDS: 384.02
        CONFIG_FILE: './config/benchmark-config.xml'
      - target: javadoc
        BASELINE_SECONDS: 530
        CONFIG_FILE: './config/benchmark-javadoc-config.xml'
    runs-on: ubuntu-latest
    env:
       BASELINE_SECONDS: ${{ matrix.BASELINE_SECONDS }}
       CONFIG_FILE: ${{ matrix.CONFIG_FILE }}
```

---

Done